app/src/lib/git/rebase.ts

```diff
@@ -87,6 +87,24 @@ export async function getRebaseContext(
   return null
 }
 
+/**
+ * Attempt to read the `.git/REBASE_HEAD` file inside a repository to confirm
+ * the rebase is still active.
+ */
+async function readRebaseHead(repository: Repository): Promise<string | null> {
+  try {
+    const rebaseHead = Path.join(repository.path, '.git', 'REBASE_HEAD')
+    const rebaseCurrentCommitOutput = await FSE.readFile(rebaseHead, 'utf8')
```

Comment:
do we care about the contents of `REBASE_HEAD`? i see you're returning it but the docs say we just want to see if its there. this could just be [`FSE.pathExists`](https://github.com/jprichardson/node-fs-extra/blob/HEAD/docs/pathExists.md) but maybe attempting to read the file gives us more confidence that a valid rebase is in progress?

Reply:
I use it in `continueRebase` to track when the in-flight rebase skips a commit (because no tracked changes):

```
    log.warn(
      `[rebase] no tracked changes to commit for ${rebaseCurrentCommit.trim()}, continuing rebase but skipping this commit`
      `[rebase] no tracked changes to commit for ${rebaseCurrentCommit}, continuing rebase but skipping this commit`
    )
```